activerecord/test/cases/adapters/postgresql/range_test.rb

```diff
@@ -196,14 +196,14 @@ def test_timezone_awareness_endless_tzrange
       time_string = Time.current.to_s
       time = Time.zone.parse(time_string)
 
-      record = PostgresqlRange.new(tstz_range: time_string...)
-      assert_equal time..., record.tstz_range
+      record = PostgresqlRange.new(tstz_range: time_string..)
```

Comment:
This just aligns with the next test, which [uses inclusion](https://github.com/rails/rails/blob/2b9c25c2c432572ed7ad19273e1a7a894e9fd8cc/activerecord/test/cases/adapters/postgresql/range_test.rb#L219).